mesa/examples/basic/schelling/model.py

```diff
@@ -13,7 +13,7 @@ def __init__(
         width: int = 40,
         density: float = 0.8,
         minority_pc: float = 0.5,
-        homophily: int = 3,
+        homophily_ratio: float = 0.3,
```

Comment:
why 0.3? It would make more sense to go to 3/8 (8 is the neighborhood size of a Moore grid with a radius of 1 and all fields occupied.

Reply:
Thank you for reviewing. I will change 0.3 to 3/8 (0.375).

Reply:
@quaquel The step size of the slider is 0.1. So the ratio can either be 0.3 or 0.4. 
If the ratio is 0.3, agents with 3 similar neighbors and 8 total neighbors will be happy, as 3/8 = 0.375. 
And if the ratio is 0.4, agents with 3 similar neighbors and 8 total neighbors won't be happy.
So should I change 0.3 to 0.4?

Reply:
Maybe we can change the step size to 0.125 as well?